MyLibrary.Domain/Entities/Book.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MyLibrary.Domain.Entities
+{
+    public class Book
+    {
+        public int rating;
```

Comment:
Wouldnt expose a plain variable like this as you would lose control over the manipulation of this class member, so i would wrap it in a property like below and change the name to Rating to be consistent